src/app/app.module.ts

```diff
@@ -18,7 +19,7 @@ import { ParallaxHeaderComponent } from './components/parallax-header/parallax-h
         BrowserModule,
         AppRoutingModule
     ],
-    providers: [],
+    providers: [ScrollService],
```

Comment:
The service registration is unnecessary when the service is provided in the root